src/js/structs/__tests__/CompositeState-test.js

```diff
@@ -70,7 +70,7 @@ describe("CompositeState", function() {
     });
 
     it("handles null data gracefully", function() {
-      expect(CompositeState.addNodeHealth).not.toThrow();
```

Comment:
The previous call overwrote the `this`, making the method fail. I think this is the very same thing as the test wanted to test for, without using a shorthand, so I just added it.